### PR TITLE
Add code tabs for overviews/custom-collections

### PR DESCRIPTION
**Pull request by flomebul**

### CODE REVIEW

File: _overviews/core/custom-collections.md [hunk 1]
```diff
@@ -77,6 +83,49 @@ class Capped1[A] private (val capacity: Int, val length: Int, offset: Int, elems
...
+  def iterator: Iterator[A] = new AbstractIterator[A]:
+    private var current = 0
+    def hasNext = current < self.length
+    def next(): A =
+      val elem = self(current)
+      current += 1
+      elem
+
+  override def className = "Capped1"
+
```
Comment (bishabosha) [inline _overviews/core/custom-collections.md:124]: I think for this example an end marker would be welcome
```suggestion
end Capped1
```

File: _overviews/core/custom-collections.md [hunk 2]
```diff
@@ -383,51 +561,104 @@ representation.
...
-    ) extends IndexedSeq[Base]
-      with IndexedSeqOps[Base, IndexedSeq, RNA1] {
+final class RNA1 private (
+  val groups: Array[Int],
+  val length: Int
+) extends IndexedSeq[Base]
+  with IndexedSeqOps[Base, IndexedSeq, RNA1] {
 
-      import RNA1._
+  import RNA1._
```
Comment (bishabosha) [inline _overviews/core/custom-collections.md:576]:
```suggestion
  import RNA1.*
```

File: _overviews/core/custom-collections.md [hunk 3]
```diff
@@ -383,51 +561,104 @@ representation.
...
+import collection.mutable
+import collection.immutable.{ IndexedSeq, IndexedSeqOps }
+
+final class RNA1 private
+( val groups: Array[Int],
+  val length: Int
+) extends IndexedSeq[Base],
+  IndexedSeqOps[Base, IndexedSeq, RNA1]:
+
+  import RNA1._
```
Comment (bishabosha) [inline _overviews/core/custom-collections.md:625]:
```suggestion
  import RNA1.*
```

File: _overviews/core/custom-collections.md [hunk 4]
```diff
@@ -576,38 +837,84 @@ this is why we always get a `Vector` as a result.
...
+{% tabs RNA2_1 class=tabs-scala-version %}
+{% tab 'Scala 2' for=RNA2_1 %}
+~~~ scala
+import scala.collection.{ View, mutable }
+import scala.collection.immutable.{ IndexedSeq, IndexedSeqOps }
+
+final class RNA2 private (val groups: Array[Int], val length: Int)
+  extends IndexedSeq[Base] with IndexedSeqOps[Base, IndexedSeq, RNA2] {
+
+  import RNA2._
```
Comment (bishabosha) [inline _overviews/core/custom-collections.md:849]:
```suggestion
  import RNA2.*
```

File: _overviews/core/custom-collections.md [hunk 5]
```diff
@@ -576,38 +837,84 @@ this is why we always get a `Vector` as a result.
...
+{% endtab %}
+{% tab 'Scala 3' for=RNA2_1 %}
+~~~ scala
+import scala.collection.{ View, mutable }
+import scala.collection.immutable.{ IndexedSeq, IndexedSeqOps }
+
+final class RNA2 private (val groups: Array[Int], val length: Int)
+  extends IndexedSeq[Base], IndexedSeqOps[Base, IndexedSeq, RNA2]:
+
+  import RNA2._
```
Comment (bishabosha) [inline _overviews/core/custom-collections.md:887]:
```suggestion
  import RNA2.*
```

**Review by bishabosha — APPROVED** [review]
Thanks again!

File: _overviews/core/custom-collections.md [hunk 6]
```diff
@@ -347,16 +506,35 @@ methods (such as `iterator` in our case), if any.
...
+
+object Base {
+  val fromInt: Int => Base = Array(A, U, G, C)
+  val toInt: Base => Int = Map(A -> 0, U -> 1, G -> 2, C -> 3)
+}
+~~~
+{% endtab %}
+{% tab 'Scala 3' for=Base_1 %}
+~~~ scala
+abstract class Base
```
[the remark this answers is not in the document]
Comment (flomebul) [reply _overviews/core/custom-collections.md]: I originally had the same idea, but seeing the text after this example, I thought it best to keep it as it is: an additional example of using the collection.

File: _overviews/core/custom-collections.md (hunk 5 above)
Comment (flomebul) [reply _overviews/core/custom-collections.md]: It look like I forget a lot of stars... Thanks

**Pull request merged**